au/math.hh

```diff
@@ -318,31 +318,41 @@ constexpr bool isnan(QuantityPoint<U, R> p) {
     return std::isnan(p.in(U{}));
 }
 
+namespace detail {
+// Some compilers do not support lambdas in constexpr contexts, so we make a manual function object.
```

Comment:
Should we update the comment about this being a C++17 thing, or is it the case that even with C++17, some compilers still don't support it.

Reply:
Good point!  Addressed in #252.